changelog/12126.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:bug
+auth/secret: Adding capability to use comma separated parameters for auth enable/tune
```

Comment:
I don't see this multiline syntax documented anywhere so I wouldn't assume it's valid.

When choosing the string to put before the colon, have a look at https://github.com/hashicorp/vault/blob/master/CHANGELOG.md.  If you don't see any examples of it, it's probably not the right value.  I'm not sure what I'd put here, maybe `api, cli:`?

Reply:
Sounds good! I will follow those then.
I think cli/api is a good choice for this PR.